include/staff/ticket-view.inc.php

```diff
@@ -836,9 +836,10 @@ class="icon-building icon-fixed-width"></i> <?php
                                  $e->getId(),
                                  Format::htmlchars($e->getAddress()));
                      }
+                     $staffDepts = array_keys($thisstaff->getDepartmentNames(true));
```

Comment:
Why not use `$thisstaff->getDepts()` ? It's cached and already in the format you need.

Reply:
Updated! I had done it like that somewhere else too. I went ahead and corrected that one as well. Let me know if I should correct it separately instead though.